editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -336,8 +337,11 @@ export const ConditionalSection = React.memo(({ paths }: { paths: ElementPath[]
 
   return (
     <div style={{ paddingBottom: 8 }} data-testid={ConditionalSectionTestId}>
-      <InspectorSubsectionHeader
+      <FlexRow
```

Comment:
This change will also need to be made in https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/inspector/sections/code-element-section.tsx